src/RTCRtpTransceiver.ts

```diff
@@ -17,13 +16,13 @@ export default class RTCRtpTransceiver {
     _stopped: boolean;
 
     constructor(args: {
-        peerConnectionId: number,
-        isStopped: boolean,
-        direction: string,
-        currentDirection: string,
-        mid?: string,
-        sender: RTCRtpSender,
-        receiver: RTCRtpReceiver,
+        peerConnectionId: number;
```

Comment:
Unsure why you've changed so much formatting 🤔 this is no longer functional.

Reply:
Oo, this is formatted by my local prettier configuration, but it is functional on my end

Reply:
Yes, undo all formatting please.